src/components/screen-header/LargeScreenHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import {ThemeText} from '@atb/components/text';
+import {GlobalMessage, GlobalMessageContextType} from '@atb/global-messages';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {getStaticColor, StaticColor} from '@atb/theme/colors';
+import useFocusOnLoad from '@atb/utils/use-focus-on-load';
+import React from 'react';
+import {View, ViewStyle} from 'react-native';
+import {ContentWithDisappearingHeader} from '../disappearing-header';
+import {LargeHeaderButton, LargeHeaderButtonProps} from './HeaderButton';
+
+export {AnimatedScreenHeader} from './AnimatedScreenHeader';
+
+export type LargeScreenHeaderProps = {
+  title?: string;
+  titleA11yLabel?: string;
+  /**
+   * For specifying the alert context for alerts that should be shown in this
+   * header. If no context is specified then no alerts are shown.
+   */
+  globalMessageContext?: GlobalMessageContextType;
+  style?: ViewStyle;
+  containerStyle?: ViewStyle;
```

Comment:
Will both these parameters be necessary? Can we skip one, or maybe even both? 🤔

Reply:
I though it would be good to support most of the params from the current header, but looking at how that's used, it doesn't seem necessary. Removing them both!